bin/compile

```diff
@@ -147,6 +147,10 @@ export LIBRARY_PATH=/app/.heroku/vendor/lib:$BUILD_DIR/.heroku/vendor/lib:/app/.
 export LD_LIBRARY_PATH=/app/.heroku/vendor/lib:$BUILD_DIR/.heroku/vendor/lib:/app/.heroku/python/lib:$LD_LIBRARY_PATH
 export PKG_CONFIG_PATH=/app/.heroku/vendor/lib/pkg-config:$BUILD_DIR/.heroku/vendor/lib/pkg-config:/app/.heroku/python/lib/pkg-config:$PKG_CONFIG_PATH
 
+if [[ $VCAP_SERVICES == *appdynamics* ]]; then
+  source $BIN_DIR/steps/appdynamics-setup
+fi
+
```

Comment:
Agree and the check will now be present in the start of the apdynamics-setup script.
Also this script will be sourced after Install Python in compile script